circleci/templates/base_config.yml

```diff
@@ -0,0 +1,142 @@
+version: 2.1
```

Comment:
You should add a comment here, the top line, describing what the template does. IE, what steps and workflows are defined. As we get more templates, it'll be important to have a summary at the top to avoid having to read the whole thing to figure it out.